lrrbot/commands/card.py

```diff
@@ -19,19 +22,32 @@ def set_cardview(lrrbot, conn, event, respond_to, setting):
 	lrrbot.cardview = (setting == "on")
 	conn.privmsg(respond_to, "Card viewer %s" % ("enabled" if lrrbot.cardview else "disabled"))
 
-@bot.command("card (.+)")
+@bot.command("(?:card|mtg) (.+)")
 @lrrbot.decorators.throttle(60, count=3)
-def card_lookup(lrrbot, conn, event, respond_to, search):
+def mtg_card_lookup(lrrbot, conn, event, respond_to, search):
 	"""
 	Command: !card card-name
+	Command: !mtg card-name
 	Section: misc
 
-	Show the details of a given Magic: the Gathering card.
+	Show the details of a given Magic: The Gathering card.
 	"""
-	real_card_lookup(lrrbot, conn, event, respond_to, search)
+	real_card_lookup(lrrbot, conn, event, respond_to, search, game=CARD_GAME_MTG)
 
-def real_card_lookup(lrrbot, conn, event, respond_to, search, noerror=False, includehidden=False):
-	cards = find_card(lrrbot, search, includehidden)
+@bot.command("(?:kf|keyforge) (.+)")
+@lrrbot.decorators.throttle(60, count=3)
+def keyforge_card_lookup(lrrbot, conn, event, respond_to, search):
+	"""
+	Command: !keyforge card-name
+	Command: !kf card-name
+	Section: misc
+
+	Show the details of a given KeyForge card.
+	"""
+	real_card_lookup(lrrbot, conn, event, respond_to, search, game=CARD_GAME_KEYFORGE)
+
+def real_card_lookup(lrrbot, conn, event, respond_to, search, noerror=False, includehidden=False, game=None):
```

Comment:
It makes more sense for `game` to default to `CARD_GAME_MTG` than to `None` and since every call site passes it anyway it can be mandatory. Also searching across all games is something you'd never want to do anyway.